docs/nfts.md

```diff
@@ -2,4 +2,79 @@
 title: NFTs
 ---
 
-Docs coming soon.
+### React function component example
+
+The following is a component that implements various tool examples using a previous json dump named `ConsolidatedRemarks` (with the working code equivalent commented out). In a nutshell, the component logs the winners of my contest [bishoujo.store](http://bishoujo.store) with a counter of how many times that individual emoted on the nft.
```

Comment:
You can't refer to personal pronouns in docs. "My" has no meaning here. Remove personal angle please. 

Also, due to the evolving nature of external projects, I'd prefer if we didn't link to them from docs and risk making the content outdated asap. If you're documenting an example, if should be atomic, standalone, encapsulated, not depend on much else.